compute/admin_guide/shared/PCC_Azure_Agentless.json

```diff
@@ -0,0 +1,54 @@
+{
```

Comment:
Hi @rcaballeromx is this a new file/folder in the AG?

I do not think we should add this to the book. I can publish this json on docs.paloalto as a file and you can add a URL in the adoc file for it. Let me know if this works

Reply:
@Pubs-MV I didn't add the file to the book. I added the file on a shared folder but didn't include it in the book.
You can download the file and save it where it needs to be.
I will change the link so it is available for download.

Reply:
please delete this topic and add a link from your workflow to https://redlock-public.s3.amazonaws.com/azure/azureAgentlessPermissions.json